test/connector/tcp/pg/pkg/run_test_case.go

```diff
@@ -66,3 +66,43 @@ func RunQuery(
 
 	return string(cmdOut), err
 }
+
+// RunQuery runs a simply test query for the given client configuration and port.
+func RunJDBCQuery(
+	clientConfig testutil.ClientConfiguration,
+	connectPort testutil.ConnectionPort,
+) (string, error) {
+	const jdbcJARPath = "/secretless/test/util/jdbc/jdbc.jar"
+
+	args := []string{
+		"-jar", jdbcJARPath,
+		"-d", "postgres",
+		"-m", "postgresql",
+		"-h", fmt.Sprintf("%s:%d", connectPort.Host(), connectPort.Port),
+		"-U", clientConfig.Username,
+		"-P", clientConfig.Password,
+		"select count(*) from test.test",
+	}
+
+	// Pre command logs
+	convey.Println("")
+	convey.Println("---<< EXECUTED")
+
+	cmdOut, err := exec.Command("java", args...).CombinedOutput()
+	fmt.Println(args)
```

Comment:
I would think that we want to show the args before things get executed otherwise if there a problem this line won't get reached.